cmd/kpod/save.go

```diff
@@ -31,6 +33,10 @@ var (
 			Name:  "quiet, q",
 			Usage: "Suppress the output",
 		},
+		cli.BoolFlag{
+			Name:  "oci",
+			Usage: "Save image to oci-archive, default is docker-archive",
```

Comment:
I am thinking we should change this.  This should probably be a configuration option, and then have a flag that allows you to specify the output type

--format docker-archive
--format oci

Eventually I am hoping the default will be OCI, but making the tool permanently be docker-archive, seems short sited.

Reply:
Changed to use the format flag instead.